packages/relay-compiler/util/md5.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule md5
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import crypto from 'crypto';
+
+const md5 = (x: string): string => {
+  return crypto
+    .createHash('md5')
+    .update(x, 'utf8')
+    .digest('hex');
+};
+
+module.exports = md5;
```

Comment:
Btw, just out of curiosity, I was wondering why you chose md5 over e.g. sha-256? Is it because the hash is shorter and the risk of collisions is very low when the user doesn’t maliciously _try_ to craft a document that would generate a collision (which I _think_ is the status quo, but I’m no expert)?

Reply:
It's an existing hash function from the existing code base, I merely refactored it into a separate file for re-use. I'm no expert as well, but there shouldn't be any problem changing it to sha-256 if facebook wants to. I think that's a little out of scope of what this PR is about though, so I will just leave this as is for now.

Re merging and time, yes ok I understand it will take some time. However is it possible for you to approve to indicate that everything looks ok from the reviewer's perspective? Of course only do that if you do think that everything looks ok. Happy to make more changes if you see the need. @alloy . Thanks!